src/analysis/special_functions/pow.lean

```diff
@@ -991,6 +991,24 @@ by { convert tendsto_rpow_div_mul_add (1:ℝ) _ (0:ℝ) zero_ne_one, ring_nf }
 lemma tendsto_rpow_neg_div : tendsto (λ x, x ^ (-(1:ℝ) / x)) at_top (𝓝 1) :=
 by { convert tendsto_rpow_div_mul_add (-(1:ℝ)) _ (0:ℝ) zero_ne_one, ring_nf }
 
+/-- The function `(1 + t/x) ^ x` tends to `exp t` at `+∞`. -/
+lemma tendsto_one_plus_div_rpow_tendsto_exp (t : ℝ) :
```

Comment:
This lemma name is strange (remove the second `tendsto`?), and similarly for next lemma. Otherwise, LGTM.